internal/config/run_context.go

```diff
@@ -179,12 +179,18 @@ func (r *RunContext) IsInfracostComment() bool {
 }
 
 func (r *RunContext) IsCloudEnabled() bool {
-	if r.isCommentCmd && r.Config.EnableCloudForComment {
-		log.Debug("IsCloudEnabled is true for comment with org level setting enabled.")
+	if r.Config.EnableCloud != nil {
+		log.WithFields(log.Fields{"is_cloud_enabled": *r.Config.EnableCloud}).Debug("IsCloudEnabled explicitly set through Config.EnabledCloud")
```

Comment:
FYI - we prefer using the internal `logging` package over the default logrus log. This logger will have default contextual info set on it.

```suggestion
		logging.Logger.WithFields(log.Fields{"is_cloud_enabled": *r.Config.EnableCloud}).Debug("IsCloudEnabled explicitly set through Config.EnabledCloud")
```